packages/frontend/src/components/common/Filters/FilterRuleForm.tsx

```diff
@@ -88,28 +88,30 @@ const FilterRuleForm: FC<Props> = ({
                         }}
                     />
 
-                    <HTMLSelect
-                        className={!isEditMode ? 'disabled-filter' : ''}
-                        fill={false}
+                    <Select
+                        size="xs"
+                        w="130px"
```

Comment:
value is cutoff 
<img width="140" alt="Screenshot 2023-10-13 at 10 23 20" src="https://github.com/lightdash/lightdash/assets/9117144/ca834738-bef9-4de0-b091-d0d9e70b2825">